peps/pep-0760.rst

```diff
@@ -0,0 +1,162 @@
+PEP: 760
+Title: No More Bare Excepts
+Author: Pablo Galindo <pablogsal@python.org>, Brett Cannon <brett@python.org>
+PEP-Delegate: TBD
+Status: Draft
+Type: Standards Track
+Created: 30-Sep-2024
```

Comment:
The date of assigning the PEP number
```suggestion
Created: 2-Oct-2024
```

Reply:
Actually, days must be two digits, so `02-Oct-2024` (I just found this out with PEP 759).